internal/cluster/validator.go

```diff
@@ -485,31 +483,7 @@ func (v *clusterValidator) printNetworkPrefixValid(c *clusterPreprocessContext,
 }
 
 func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) ValidationStatus {
-	var min int64
-	var max int64
-	for _, h := range c.cluster.Hosts {
-		var inventory models.Inventory
-		if err := json.Unmarshal([]byte(h.Inventory), &inventory); err != nil {
-			v.log.WithError(err).Warnf("Illegal inventory for host %s", h.ID.String())
-			continue
-		}
-		if inventory.Timestamp == 0 || *h.Status == models.HostStatusDisconnected ||
-			*h.Status == models.HostStatusDisabled || *h.Status == models.HostStatusResettingPendingUserAction ||
-			*h.Status == models.HostStatusDiscovering {
-			continue
-		}
-
-		if inventory.Timestamp < min || min == 0 {
-			min = inventory.Timestamp
-		}
-		if inventory.Timestamp > max {
```

Comment:
we shouldn't delete it. It is not enough to verify that clock is synced. We just saw that case that all the clocks were synced but with different ntp servers and timediff was very big between them